src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -327,6 +327,15 @@ private function addProfilerSection(ArrayNodeDefinition $rootNode)
                         ->booleanNode('only_main_requests')->defaultFalse()->end()
                         ->scalarNode('dsn')->defaultValue('file:%kernel.cache_dir%/profiler')->end()
                         ->booleanNode('collect_serializer_data')->info('Enables the serializer data collector and profiler panel')->defaultFalse()->end()
+                        ->arrayNode('cli')
+                            ->canBeEnabled()
+                            ->children()
+                                ->arrayNode('excludes')
+                                    ->defaultValue(['cache:clear', 'messenger:consume'])
+                                    ->info('Exclude commands with names or namespaces')
```

Comment:
Prefixes instead of namespaces?

Reply:
Not sure about this suggestion,"namespace" seems more in fit with command naming (see `bin/console list --help`).